tests/apm/mod.rs

```diff
@@ -54,7 +54,7 @@ fn read_first_non_monitor_line(file: &mut BufReader<&File>, line: &mut String) {
 
 #[test]
 fn logging() {
-    let _ = fs::remove_file("test_apm_log.txt");
+    let _ = fs::remove_file("test_log.txt");
```

Comment:
For some reason the old name of this file didn't match the name I put in the .gitignore way back when this test was created, so I fixed that.